lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -129,7 +129,7 @@ def create_database(self, database: str, user: str) -> None:
                     sql.Identifier(database)
                 )
             )
-            for user_to_grant_access in [user] + self.system_users:
+            for user_to_grant_access in [user, "admin"] + self.system_users:
```

Comment:
Is this line needed if we add `pg_read_all_data, pg_write_all_data` to the role?

Reply:
It's still needed after we simplify the privileges to only those two. At least the `CONNECT` privilege is needed (which is granted through this block).